client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
It should be up to the user how he obtains su permissions IMHO. And he should do it knowingly. I would consider a script that runs `sudo` (especially without my knowledge) as malicious software. Moreover, some people do not have `sudo` installed.